src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/config/WebClientConfiguration.kt

```diff
@@ -144,7 +148,7 @@ class WebClientConfiguration(
 
```

Comment:
You could aslo add this into the WebClients in `tokenWebClient` and `getWebClient`

Reply:
Done